From: Paul Boddie <paul@boddie.org.uk>
Date: Tue, 15 Aug 2023 23:50:33 +0200
Subject: Removed SVG and wikidraw from wikiconfig.

The drawing-related modules are not yet packaged and should be optional,
anyway, so this patch comments them out in the wikiconfig.

moin (2.0+git20230330.c55318e3-1) unstable; urgency=medium

  * Initial release.
---
 src/moin/config/wikiconfig.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/moin/config/wikiconfig.py b/src/moin/config/wikiconfig.py
index 470508c..dc2f430 100644
--- a/src/moin/config/wikiconfig.py
+++ b/src/moin/config/wikiconfig.py
@@ -153,7 +153,7 @@ class Config(DefaultConfig):
         'font_awesome',
         'ckeditor',
         'autosize',
-        'svgedit_moin', 'twikidraw_moin', 'anywikidraw',
+        #'svgedit_moin', 'twikidraw_moin', 'anywikidraw',
         'jquery_tablesorter',
         'pygments',
     ]
